### PR TITLE
Updated CPed,CVehicle,CWeaponInfo

### PR DESCRIPTION
**Pull request by altforfr**
Updated CPed,CVehicle,CWeaponInfo

### CODE REVIEW

**Comment by Yimura** [comment]
You're classes are completely wrong.

**Comment by Yimura** [comment]
Take a look at the below class `rage::fwEntity`:
https://github.com/Yimura/GTAV-Classes/blob/e0b6b57b25819ef7edc2dd27765cbc80191a4d77/entities/fwEntity.hpp#L17-L25

**Comment by altforfr** [comment]
Ah so thats why

**Comment by altforfr** [comment]
Makes sense I can close pull request if u want cuz only things that was important was the draw handlers and model info

**Comment by Yimura** [comment]
Please take another look at inheritance and please edit these files locally, that way you can check if they're correct or not (static_asserts). Don't use in-browser editing of your repository.

Maybe next time create an issue on what you're planning to do in the future or join my [discord server](https://discord.gg/EG4zHFR) if you wish to speak directly to me and others that contribute.

**Comment by Yimura** [comment]
> Your classes are completely wrong.

They aren't completely wrong though, I see some new stuff that might be nice to add.

**Comment by Aure7138** [comment]
there is no rage::CPhysical in your commit

**Comment by Yimura** [comment]
> there is no rage::CPhysical in your commit

The repository already has those.

**Comment by Aure7138** [comment]
My mistake, didn't check, but if you want to update the CPed class, it should start after 0x30C

File: ped/CPed.hpp [hunk 1]
```diff
@@ -16,18 +17,27 @@
...
-	class CVehicle *m_vehicle; //0x0D30
-	char pad_0D38[912]; //0x0D38
-	class CPlayerInfo *m_player_info; //0x10C8
-	class CPedInventory* m_inventory; //0x10D0
-	class CPedWeaponManager *m_weapon_manager; //0x10D8
-	char pad_10E0[907]; //0x10E0
-	uint8_t m_ped_task_flag; //0x146B
-	char pad_146C[196]; //0x146C
-	float m_armor; //0x1530
+	CPedModelinfo* m_model_info; //0x0020
```
Comment (Aure7138) [inline ped/CPed.hpp:20]: If you inherit from rage::CPhysical, here CPedModelinfo* m_model_info; //0x0020 is not the location of 0x20, but 0x30C

**Pull request closed without merging**